### ISSUE TITLE
ContrastRatio Problem in Settings UI and Terminal UI

### ISSUE THREAD

**Issue by W0n9** [Eugeny/tabby#8099]
**Describe the problem**:
After update tabby 1.0.193, I found a problem that there is Contrast Ratio problem occurred, which lead to difficully to read.

Here are some examples:
![微信截图_20230320100306](https://user-images.githubusercontent.com/38560218/226233778-f76eb489-9c5e-4585-8ef2-1a12d95262ba.png)
Windows Terminal vs Tabby
![微信截图_20230320100701](https://user-images.githubusercontent.com/38560218/226233827-e2b5f6b4-af25-4240-8993-f56e2c4327dd.png)

**Comment by charley008** [comment]
![2023-03-20 130103](https://user-images.githubusercontent.com/24822683/226251017-396795ff-6457-4fbc-b117-00e67f9d112c.png)
![2023-03-20 125957](https://user-images.githubusercontent.com/24822683/226251024-d0153bd3-4fae-4e0b-81a6-3b7634beaa67.png)

**Comment by Eugeny** [comment]
Thanks, which color scheme are you using?

**Comment by W0n9** [comment]
> Thanks, which color scheme are you using? As a temporary workaround, you can switch back to the old window theme in Settings -> Window

I use the default theme, here is my config file about color scheme
```yaml
  colorScheme:
    name: Material
    foreground: '#eceff1'
    background: rgba(38, 50, 56, 1)
    selection: null
    cursor: '#FFCC00'
    cursorAccent: null
    colors:
      - '#000000'
      - '#D62341'
      - '#9ECE58'
      - '#FAED70'
      - '#396FE2'
      - '#BB80B3'
      - '#2DDAFD'
      - '#d0d0d0'
      - rgba(255, 255, 255, 0.2)
      - '#FF5370'
      - '#C3E88D'
      - '#FFCB6B'
      - '#82AAFF'
      - '#C792EA'
      - '#89DDFF'
      - '#ffffff'
  customColorSchemes: []
```

And switch back to the Standard(legacy) window theme Looks good to me!👍

**Comment by fenglangjunzi** [comment]
color scheme: Atom
Settings -> Window -> Follow the color scheme

<img width="833" alt="image" src="https://user-images.githubusercontent.com/5000975/226502151-ed2be201-2037-4450-9734-0da2f23970fd.png">

**Comment by fenglangjunzi** [comment]
SSH Search：command + F

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/5000975/226538600-80de427b-99cf-4193-a774-16f25b9d6ab5.png">

**Comment by Eugeny** [comment]
Should be fixed in 1.0.196 now

**Comment by yume-chan** [comment]
I'm on 1.0.196, the contrast ratio was improved comparing with screenshots above, but still too low to see some texts.

![image](https://user-images.githubusercontent.com/1330321/233762175-6cd21c73-3707-4fdb-bd54-b6f7538ec0c5.png)

![image](https://user-images.githubusercontent.com/1330321/233762216-68322462-b7ff-4a07-a116-aecb454af75d.png)

![image](https://user-images.githubusercontent.com/1330321/233762253-97df1e78-f11b-4d95-97eb-a84ded8232cf.png)

Notice the selected item in radio button group is not highlighted.

![image](https://user-images.githubusercontent.com/1330321/233762326-1d33535f-107b-4767-9db7-1ed2ee8a5096.png)
